themes/admin/Sea_Green/css/sea_green-rtl.css

```
@@ -18264,6 +18303,11 @@ ol.breadcrumb.title-bar-breadcrumb li.active {
     display: block !important;
   }
 }
+i.view-switch:focus {
+  outline: 2px solid currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)